include/z64actor.h

```diff
@@ -357,4 +357,101 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    NAVI_ENEMY_DEFAULT,
+    NAVI_ENEMY_GOHMA,
+    NAVI_ENEMY_ENGOMA_EGG,
+    NAVI_ENEMY_ENGOMA,
+    NAVI_ENEMY_SKULLTULA_NORMAL,
+    NAVI_ENEMY_SKULLTULA_BIG,
+    NAVI_ENEMY_TAILPASARAN,
+    NAVI_ENEMY_DEKU_BABA,
+    NAVI_ENEMY_DEKU_BABA_BIG,
+    NAVI_ENEMY_DEKU_WITHERED,
+    NAVI_ENEMY_DEKU_SCRUB,
+    NAVI_ENEMY_UNUSED11,
+    NAVI_ENEMY_DODONGO,
+    NAVI_ENEMY_DODONGO2,
+    NAVI_ENEMY_DODONGO_JR,
+    NAVI_ENEMY_LIZALFOS,
+    NAVI_ENEMY_DINOLFOS,
+    NAVI_ENEMY_FIRE_KEESE,
+    NAVI_ENEMY_KEESE,
+    NAVI_ENEMY_ARMOS,
+    NAVI_ENEMY_BARINADE,
+    NAVI_ENEMY_TENTACLE,
+    NAVI_ENEMY_BUBBLE,
+    NAVI_ENEMY_BIRI,
+    NAVI_ENEMY_BARI,
+    NAVI_ENEMY_EIYER, // Stinger?
+    NAVI_ENEMY_GANON_PHANTOM,
+    NAVI_ENEMY_STALFOS,
+    NAVI_ENEMY_BUBBLE_BLUE,
+    NAVI_ENEMY_BUBBLE_WHITE,
+    NAVI_ENEMY_BUBBLE_GREEN,
+    NAVI_ENEMY_UNK31,
+    NAVI_ENEMY_UNK32,
+    NAVI_ENEMY_VOLVAGIA,
+    NAVI_ENEMY_FLARE_DANCER,
+    NAVI_ENEMY_SLUG,
+    NAVI_ENEMY_BUBBLE_RED,
+    NAVI_ENEMY_MORPHA,
+    NAVI_ENEMY_DARK_LINK,
+    NAVI_ENEMY_SHELLBLADE,
+    NAVI_ENEMY_UNK40,
+    NAVI_ENEMY_BONGO_BONGO,
+    NAVI_ENEMY_UNK42,
+    NAVI_ENEMY_GANONDORF,
+    NAVI_ENEMY_UNUSED44,
+    NAVI_ENEMY_UNK45,
+    NAVI_ENEMY_DEADHANDS_HAND,
+    NAVI_ENEMY_DEADHAND,
+    NAVI_ENEMY_WALLMASTER,
+    NAVI_ENEMY_FLOORMASTER,
+    NAVI_ENEMY_KOUME,
+    NAVI_ENEMY_KOTAKE,
+    NAVI_ENEMY_IRON_KNUCKLE_1,
+    NAVI_ENEMY_IRON_KNUCKLE_2,
+    NAVI_ENEMY_SKULL_KID_ADULT,
+    NAVI_ENEMY_REACH_REACH,
+    NAVI_ENEMY_UNUSED56,
+    NAVI_ENEMY_BEAMOS,
+    NAVI_ENEMY_ANUBIS,
+    NAVI_ENEMY_FREEZARD,
+    NAVI_ENEMY_UNUSED60,
+    NAVI_ENEMY_GANON,
+    NAVI_ENEMY_GANON2,
+    NAVI_ENEMY_UNK63,
+    NAVI_ENEMY_SKULL_KID_NO_MASK,
+    NAVI_ENEMY_SKULL_KID_WITH_MASK,
+    NAVI_ENEMY_OKUTA,
+    NAVI_ENEMY_UNK67,
+    NAVI_ENEMY_UNK68,
+    NAVI_ENEMY_TEKTITE,
+    NAVI_ENEMY_TEKTITE_BLUE,
```

Comment:
```suggestion
    NAVI_ENEMY_OCTOROK,
    NAVI_ENEMY_POE_COMPOSER,
    NAVI_ENEMY_POE,
    NAVI_ENEMY_TEKTITE_RED,
    NAVI_ENEMY_TEKTITE_BLUE,
```
Although Red Tektite, Blue Tektite is their names so maybe it should be reversed